pkg/network/usm/tests/tracer_usm_kafka_classification_linux_test.go

```diff
@@ -24,7 +25,12 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/network/tracer"
 )
 
-func getProduceVersionTest(produceVersion int16, targetAddress, serverAddress, topicName string, dialer *net.Dialer) protocolClassificationAttributes {
+func getProduceVersionTest(version *kversion.Versions, targetAddress, serverAddress, topicName string, dialer *net.Dialer) protocolClassificationAttributes {
+	produceVersion, ok := version.LookupMaxKeyVersion(produceAPIKey)
+	if !ok {
+		panic("invalid kafka API key")
+	}
```

Comment:
require.NoError

Reply:
I changed strategy and instead now I'm making sure the tested version is actually supported